docker-compose.yml

```diff
@@ -25,3 +24,15 @@ services:
       - ${PWD}/crypt.db:/home/app/crypt/crypt.db # This will do a local database. For production you should use postgresql
       - ${PWD}/fvserver/settings.py:/home/app/crypt/fvserver/settings.py # Load in your own settings file
     restart: always
+    
+  nginx:
```

Comment:
Why are we adding this?

Reply:
Hi. I saw a mention of using the nginx proxy for convenience and put it in. I removed it. I did keep the Docker compose file in the v2 standard.